java/kafka/deployment-tracing-opentelemetry.yaml

```diff
@@ -56,12 +56,10 @@ spec:
               value: "org.apache.kafka.common.serialization.StringSerializer"
             - name: OTEL_SERVICE_NAME
               value: kafka-otel
-            - name: OTEL_TRACES_EXPORTER
-              value: jaeger
             - name: OTEL_METRICS_EXPORTER
               value: none
-            - name: OTEL_EXPORTER_JAEGER_ENDPOINT
-              value: http://my-jaeger-collector:14250
+            - name: OTEL_EXPORTER_OTLP_ENDPOINT
+              value: http://my-jaeger-collector:4317
```

Comment:
I would use `http://my-otlp-endpoint:4317` everywhere. It makes clear that it's an OTLP endpoint needed, not Jaeger (even if Jaeger backend exposes anyway an OTLP endpoint).

Reply:
@ppatierno Changed to `http://my-otlp-endpoint:4317`.
It was tested and working.

Reply:
@ppatierno See above from Monday. Thanks

Reply:
Great, I missed this comment! Thanks!